app/stores/SettingsStore.js

```diff
@@ -257,7 +257,7 @@ class SettingsStore {
         //       set if not present on loading
         let support_v3_until = new Date("2018-10-20T00:00:00Z");
 
-        if (!ss.get("settings_v4") && new Date() < support_v3_until) {
+        if (!ss.has("settings_v4") && new Date() < support_v3_until) {
```

Comment:
Interesting, did not know that method. Will try that out to see the difference, thanks

Reply:
By default get returns an object, that's why your previous check always evaluated to false. !{} is always false, regardless of the number of keys inside.

Reply:
Ah cool. I thought I tried and saw that there is undefined ... Yours makes more sense and is the precise question this line of code wants to ask. Thanks for clarifying.